Resources/Prototypes/_RMC14/Roles/Jobs/Other/correspondent.yml

```diff
@@ -20,6 +20,10 @@
   special:
   - !type:AddComponentSpecial
     components:
+    - type: NpcFactionMember
+      factions:
+      - Civilian
```

Comment:
Correspondent isn't supposed to have any faction beyond UNMC from what I can tell